src/odemis/odemisd/start.py

```diff
@@ -722,8 +755,43 @@ def main(args):
             else:
                 # Return when the GUI is done
                 logging.info("Starting the GUI...")
-                subprocess.check_call(["odemis-gui", "--log-level", odemis_config["LOGLEVEL"]])
 
+                gui_start_time = time.time()
+                try:
+                    subprocess.check_call(["odemis-gui", "--log-level", odemis_config["LOGLEVEL"]])
+                except Exception as error:
```

Comment:
@pieleric Do you have any exceptions in mind? Otherwise we can use the updated version which focuses only on subprocess.CalledProcessError (which basically is all the errors from the sub process so that should cover a lot.)